### ISSUE TITLE
how to use with test demo?

### ISSUE THREAD

**Issue by thunderlzy** [facebookresearch/TimeSformer#49]
hi, i test with test_net.py and it load data with test.csv, if I want to test with local singe video and get the predict label for the video，Is there a test demo available？thank you .

**Comment by gberta** [comment]
I haven't written a demo file. You could either just put a single video in your test.csv file, and extract predictions that way. 

Alternatively, you could use the following code snippet:

```python
import torch
from timesformer.models.vit import TimeSformer

model = TimeSformer(img_size=224, num_classes=400, num_frames=8, attention_type='divided_space_time',  pretrained_model='/path/to/pretrained/model.pyth')

dummy_video = torch.randn(2, 3, 8, 224, 224) # (batch x channels x frames x height x width)

pred = model(dummy_video,) # (2, 400)
```

You would need to modify this code snippet to load your video, and also to do all the necessary data pre-processing, which you can find in dataset/kinetics.py.

**Comment by thunderlzy** [comment]
thank you

**Comment by XinyueZ** [comment]
@gberta must I convert my video to size 224 or 448 when calling` TimeSformer()` ?
In all the other arguments, are there restrictions? 
I just got crashed while running the above code.
